application/core/mvvm/library/src/main/java/com/stepango/blockme/core/mvvm/library/ui/BaseViewBindingFragment.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2023 tinkoff.ru
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stepango.blockme.core.mvvm.library.ui
+
+import android.content.Context
+import androidx.annotation.LayoutRes
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import com.stepango.blockme.core.mvvm.library.lifecycle.ViewModelFactory
+import javax.inject.Inject
+import kotlin.reflect.KClass
+
+abstract class BaseViewBindingFragment(
```

Comment:
Copied from BaseFragment but without generic type and without using `DataBindingUtils`